src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -94,7 +94,7 @@ public UIElement(string htmlTag, bool isSvg)
 			var type = GetType();
 
 			Handle = GCHandle.ToIntPtr(_gcHandle);
-			HtmlId = Handle;
+			HtmlId = $"uno-{Handle}";
```

Comment:
I'm really wondering why there is both `Handle` and `HtmlId` if they are the same? The properties are readonly and can't be changed after constructor execution, so they should always remain the same. What was the point of having both of them?